asllib/doc/doclint.py

```diff
@@ -45,18 +45,33 @@ def check_hyperlinks_and_hypertargets():
     missing_hyperlinks = hypertarget_labels.difference(hyperlink_labels)
     if not missing_hyperlinks == set():
```

Comment:
I think it's more "Pythonic" to write:
```suggestion
    if missing_hyperlinks:
```
similarly elsewhere